cds_migrator_kit/rdm/migration/transform/xml_processing/rules/summer_student_report.py

```diff
@@ -83,6 +83,7 @@ def corporate_author(self, key, value):
             departments.append(department)
         self["custom_fields"]["cern:departments"] = departments
         raise IgnoreKey("contributors")
```

Comment:
this is redundant no?

Reply:
it was creating the None values, so no it is not redundant :) without this you get standard implicit `return None` behaviour